scripts/gen.py

```diff
@@ -870,17 +879,23 @@ def is_tensor_api(fndef):
   return m is not None, fndef
 
 
+def create_funcdef(fndef, jdata):
+  fields = json.loads(jdata)
+  return FuncDef(cpp_sig=fndef, aten_sig=fields['schema'], leaf=fields['compound'] == '0')
```

Comment:
I feel it is more robust not to assume presence of given metadata.
So maybe this?
```Python
fileds.get('compound', '0') == '0'
```